testing/font-comic-neue/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=font-comic-neue
+_pkgname=comicneue
+pkgver=0_git20160917
+pkgrel=0
+pkgdesc="Comic Neue is a font that fixes the shortcomings of Comic Sans"
+url="http://comicneue.com/"
+arch="noarch"
+license="OFL-1.1"
+depends="fontconfig"
+_gitrev="216e1cca320e7e7886c9523f9afbb7972eb7a9f2"
+source="$_pkgname-$pkgver.zip::https://github.com/crozynski/comicneue/archive/$_gitrev.zip"
```

Comment:
upstream actually [tags releases](https://github.com/crozynski/comicneue/releases) could you please use the latest release instead of checking out a particular git commit?